### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/multipath-tools/commit/ba867eee7bd47ce1fce4bd76a66c4221ad166b20#commitcomment-79879438

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/multipath-tools/commit/8fca965edd6a13e499d47d7b8d34a25ce31e1bc6

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: .github/workflows/foreign.yaml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
       - name: enable foreign arch
         run: sudo docker run --rm --privileged multiarch/qemu-user-static:register --reset
       - name: run tests
-        # Github actions doesn't support referencing docker images with
+        # GitHub actions doesn't support referencing docker images with
```
Comment (jsoref) [inline .github/workflows/foreign.yaml:59]: Brand

File: libmpathvalid/mpath_valid.h [hunk 2]
```diff
@@ -20,7 +20,7 @@
 #ifndef LIB_MPATH_VALID_H
 #define LIB_MPATH_VALID_H
 
-#ifdef __cpluscplus
+#ifdef __cplusplus
```
Comment (jsoref) [inline libmpathvalid/mpath_valid.h:23]: > **Warning**
> This is a notable change

File: libmultipath/nvme/linux/nvme.h [hunk 3]
```diff
@@ -1035,7 +1035,7 @@ enum {
 	NVME_SANITIZE_LOG_STATUS_MASK		= 0x0007,
 	NVME_SANITIZE_LOG_NEVER_SANITIZED	= 0x0000,
 	NVME_SANITIZE_LOG_COMPLETED_SUCCESS	= 0x0001,
-	NVME_SANITIZE_LOG_IN_PROGESS		= 0x0002,
+	NVME_SANITIZE_LOG_IN_PROGRESS		= 0x0002,
```
Comment (jsoref) [inline libmultipath/nvme/linux/nvme.h:1038]: It's possible this comes from an upstream and thus can't be changed here.

File: libmultipath/prioritizers/alua_spc3.h [hunk 4]
```diff
@@ -143,7 +143,7 @@ inquiry_data_get_tpgs(struct inquiry_data *id)
  *-----------------------------------------------------------------------------
  */
 #define CODESET_BINARY			0x1
-#define CODESET_ACSII			0x2
+#define CODESET_ASCII			0x2
```
Comment (jsoref) [inline libmultipath/prioritizers/alua_spc3.h:146]: API break?

File: libmultipath/nvme/linux/nvme.h (hunk 3 above)
Comment (mwilck) [reply libmultipath/nvme/linux/nvme.h:1038]: It comes from an upstream, but we can change it nonetheless. The symbol isn't used in our code. Some day we should switch to git submodules.

File: libmultipath/prioritizers/alua_spc3.h (hunk 4 above)
Comment (mwilck) [reply libmultipath/prioritizers/alua_spc3.h:146]: No. This is an internal header. The symbol is unused.

**Review by mwilck — APPROVED** [review]
Thanks for doing this. I should add this action to my staging repo.

I usually ask people submitting PRs here to post their changes on the dm-devel mailing list. But I believe the list shouldn't be bothered with this. I'm going to squash these changes and pull them into https://github.com/openSUSE/multipath-tools/tree/queue.

**Comment by mwilck** [comment]
@cvaroqui , please don't merge. It will be part of my next pull request.

**Comment by xosevp** [comment]
guys, be careful. Some patches are for files from external  projects (nvme, valgrind)

**Comment by mwilck** [comment]
> guys, be careful. Some patches are for files from external projects (nvme, valgrind)

So what? We've copied these into our tree, the license allowed that.

**Pull request closed without merging**